README.md

```diff
@@ -50,14 +54,12 @@ ground up.
 
 ## Getting Started
 
-Please see our [user
-guide](https://zcash.readthedocs.io/en/latest/rtd_pages/rtd_docs/user_guide.html)
-for instructions on joining the main Zcash network.
+Please see our [user documentation](user.md) for instructions on joining the main Zcash network.
+
+<!-- Relative link destinations like `user.md` live under `./doc/book/src`. -->
```

Comment:
It would be possible to make this link Just Work, at least on a Unix checkout:
1. Move all book content into a subdirectory, say `guide`.
2. Symlink `guide` to `doc/book/src/guide`.